src/lib/internal_functions.h

```diff
@@ -110,7 +110,7 @@ static bool CompareOOVPAToAddress(iXbSymbolContext* pContext, OOVPA* Oovpa, memp
 // Return if the given (XRef'erenced) is not set yet.
 static inline bool internal_IsXRefUnset(uint16_t XRef)
 {
-    return (XRef == (uint16_t)XREF_ADDR_UNDETERMINED) || (XRef == XREF_ADDR_DERIVE);
+    return (XRef == XREF_UNDETERMINED) || (XRef <= XREF_KT_COUNT);
```

Comment:
Why is the second test changed?

Reply:
`internal_IsXRefUnset` is intended to check if xref is set for database's symbols that are grouped together. Plus cannot have kernel thunk references from the database's symbols. So, that's consider as not set. There's two difference between `internal_IsXRefUnset` and `internal_IsXRefAddrUnset`. `internal_IsXRefAddrUnset` is designed for xref_database's array that is cached.